src/ui/atoms/button/Button.css

```diff
@@ -126,6 +126,10 @@
   height: 32px;
   padding: 0 var(--sp-sm);
 }
+.Button--mds {
```

Comment:
We should adjust all the button sizes down or up one level vs creating a new `mds` size. So I guess I'd make this new size `md` and bump `lg` to `xl` and so on.